src/internal/polyfills/Number.MIN_SAFE_INTEGER.js

```diff
@@ -1,6 +1,3 @@
-import { or } from 'ramda';
-
-
-const MIN_SAFE_INTEGER = or(Number.MIN_SAFE_INTEGER, -(2 ** 53) - 1);
+const MIN_SAFE_INTEGER = Number.MIN_SAFE_INTEGER || -(2 ** 53) - 1;
```

Comment:
Would use parenthesis to explicitly state the associations of operands
```js
const MAX_SAFE_INTEGER = Number.MAX_SAFE_INTEGER || (-(2 ** 53) - 1)

Reply:
Tbh I'm so used to prettier purging my files of extra parens that I it doesn't bother me. Sooner or later we'll start using it for this project and they'll be gone anyway. 

Not saying I disagree with you (I'v actually been complaining about its handling of parens for ages - [see](https://github.com/prettier/prettier/issues/187#issuecomment-282741690) my comment towards the end), but using prettier is such a huge win in terms of time that it's worth the small bumps.

So doesn't bother me but happy to change if you would like.